src/objectExplorer/objectExplorerService.ts

```diff
@@ -255,7 +259,7 @@ export class ObjectExplorerService {
                 // look up saved password
                 let password = await this._connectionManager.connectionStore.lookupPassword(connectionCredentials);
                 if (!password) {
-                    let password = await this._connectionManager.connectionUI.promptForPassword();
+                    password = await this._connectionManager.connectionUI.promptForPassword();
                     if (!password) {
```

Comment:
I would strongly suggest enabling [https://palantir.github.io/tslint/rules/no-shadowed-variable/](https://palantir.github.io/tslint/rules/no-shadowed-variable/) so this doesn't happen